src/util.js

```diff
@@ -6,18 +6,20 @@ export function submitForm(form, { headers, cors, strict } = {}) {
 	let method = form.getAttribute("method");
 	method = method ? method.toUpperCase() : "GET";
 	let uri = form.getAttribute("action");
-	let payload = serializeForm(form);
+	let body;
 
 	if(method === "GET") {
```

Comment:
```suggestion

	let body;
	if(method === "GET") {
```

(though personally, I'd still prefer the `var` hoisting hack)

Reply:
But does the `var` hoisting hack work if you have two separate `var body = serializeForm()` statements?

Reply:
Yes: It essentially translates to adding `var body` to the function's very first line.

Reply:
And it would probably break if you add `var` twice, correct?

Reply:
Depends: JavaScript would essentially ignore the second `var` - but ESLint would complain. 🙂

Reply:
It's your library. I could live with either.

Reply:
Ma'am, I believe there's a misunderstanding here: Clearly this is now your library.